ruby/test/support/reporter_test_helper.rb

```diff
@@ -3,6 +3,9 @@ module ReporterTestHelper
 
   def runnable(name, failure = nil)
     runnable = Minitest::Test.new(name)
+    if defined? Minitest::Result
+      runnable = Minitest::Result.from(runnable)
```

Comment:
The main difference in 5.11 is here. Now instead of keeping all the run info inside the `Minitest::Test` instance that executed the test, it's copied inside a `Minitest::Result` instance. 

As such we need to use `Result#klass` to get the test class name.